### PR TITLE
Adding relay endpoint for claiming via slack

### PR DESCRIPTION
**Pull request by c-raj**

### CODE REVIEW

**Comment by houqp** [comment]
@jrgp should we wait for security review for this PR? or we can merge it to a develop/staging branch.

**Comment by jrgp** [comment]
@houqp not a bad idea.

File: src/iris_relay/app.py [hunk 1]
```diff
@@ -534,6 +557,7 @@ def get_relay_app(config=None):
     twilio_messages_relay = TwilioMessagesRelay(config, iclient)
     slack_authenticate = SlackAuthenticate()
     slack_messages_relay = SlackMessagesRelay(config)
+    slack_messages_relay = SlackMessagesRelay(config, iclient)
```
Comment (jrgp) [inline src/iris_relay/app.py]: hmm can you remove the duplicate slack_messages_relay line?

File: src/iris_relay/app.py [hunk 2]
```diff
@@ -483,7 +506,7 @@ def process_request(self, req, resp):
                 elif segments[2] == 'gmail' or segments[2] == 'gmail-oneclick':
                     return
         elif len(segments) == 1:
-            if segments[0] == 'health' or segments[0] == 'healthcheck':
+            if segments[0] == 'health' or segments[0] == 'healthcheck' or segments[2] == 'slack':
```
Comment (jrgp) [inline src/iris_relay/app.py]: will segments[2] exist if len(segments) == 1?

**Pull request merged**